packages/scope-manager/src/ScopeManager.ts

```diff
@@ -77,13 +76,6 @@ class ScopeManager {
   public isImpliedStrict(): boolean {
     return this.#options.impliedStrict === true;
   }
-  public isStrictModeSupported(): boolean {
-    return this.#options.ecmaVersion != null && this.#options.ecmaVersion >= 5;
-  }
-
-  public isES6(): boolean {
-    return this.#options.ecmaVersion != null && this.#options.ecmaVersion >= 6;
-  }
```

Comment:
Is there some unified scope management interface we're expected to adhere to? I couldn't find anything on GitHub/Google.

Reply:
Yeah it's this:
https://eslint.org/docs/latest/developer-guide/scope-manager-interface#scopemanager-interface

Reply:
Hmm yeah I couldn't tell if it was *necessary* to implement that or not. I'll just add these back in.

Reply:
They're not necessary as part of the documented API but it's probably safer to leave them in yeah. I've found that there is a surprising number of packages that explicitly depend on the private internals of the scope manager.